reana_server/rest/workflows.py

```diff
@@ -974,6 +974,7 @@ def get_workflow_parameters(workflow_id_or_name):  # noqa
               {
                 'id': 'dd4e93cf-e6d0-4714-a601-301ed97eec60',
                 'name': 'workflow.24',
+                'type': 'serial',
```

Comment:
Missing update in docstring with the new `type` attribute in the response `schema`, like in [`server`](https://github.com/reanahub/reana-workflow-controller/pull/153/files#diff-3ee9faf1efe690830f3d86cb5d3cf590R1326).